openlibrary/plugins/openlibrary/js/carousel/Carousel.js

```diff
@@ -108,13 +108,30 @@ const Carousel = {
                 cover.id = ocaid;
             }
 
-            const $el = $(`
-                <div class="book carousel__item">
+
+            const default_cover_url = 'https://openlibrary.org/images/icons/avatar_book-lg.png';
+
+            let book_cover = undefined
+            if (cover.id) {
+                book_cover = `
                     <div class="book-cover">
```

Comment:
Remove this div from `bookCover`.  It should be present for all cases, so add it to `$el`.
Don't forget to remove the closing div here, as well.

Reply:
Have moved both `<div class="book-cover">` and `<a href="${work.key}">` to the $el tag. (Moved the `<a href="${work.key}"` tag as was identical in both the image book_cover path and the default cover path.)